### PR TITLE
src change - for gh-pages

### PR DESCRIPTION
**Pull request by emilyville**
- In order to generate github pages, had to move everything out of the src/ and into top level.
- also added excludes for the _site/
- updated baseurl

### CODE REVIEW

**Comment by webinista** [comment]
Looks like one of the default posts is missing a valid date in the YAML front matter, so Jekyll can't build the site. There are also some merge conflicts.

@awong-dev I tried merging, but this appears to delete everything. Do you have time to take a look?

**Comment by awong-dev** [comment]
Gha... @webinista I completely missed this.  If you assign me a PR, can you possibly ping me on slack?  I get so many github notifications, I have a tendency to miss these.  If you haven't seen anything from me in like an hour, you can be pretty sure it's been lost. :(

I'll try to get better at watching my assigned queue, but always feel free to pester if I'm not responding.

**Comment by emilyville** [comment]
ok @webinista or @awong-dev all green again!
- It was pulling in a default post file from the gem - so I added an vendor exclude to fix that
- Then, the html checker was expecting a different structure
- then, the links with a "/" were throwing it off so changed to "#"
- added rake targets for better shortcuts

**Comment by emilyville** [comment]
Made the footer link correct instead of just #. :)

**Comment by emilyville** [comment]
Removed the "roadrunner" baseurl from the url to fix the external link html proofer tests.

File: Rakefile [hunk 1]
```diff
@@ -5,6 +5,16 @@ task :build do
...
 end
 
+desc "Serve the website"
+task :serve do
+  sh "bundle exec jekyll serve"
+end
+
+desc "Cleanup the website"
+task :clean do
+  sh "bundle exec jekyll clean"
```
Comment (awong-dev) [inline Rakefile]: ? I don't think this is a real jekyll command?

File: _config.yml [hunk 2]
```diff
@@ -3,18 +3,23 @@ title: Roadrunner
...
+baseurl: "/roadrunner" # the subpath of your site, e.g. /blog
+url: "https://department-of-veterans-affairs.github.io/"
 
 repourl: "https://github.com/department-of-veterans-affairs/roadrunner/"
 
 # Build settings
 markdown: kramdown
 
-destination: ./_site
+destination: ./_site/roadrunner
```
Comment (awong-dev) [inline _config.yml]: Is this correct?

**Comment by awong-dev** [comment]
@emilyville back at ya.

File: Rakefile (hunk 1 above)
Comment (emilyville) [inline Rakefile]: it basically just deletes the contents of the destination directory

http://www.rubydoc.info/github/mojombo/jekyll/Jekyll/Commands/Clean

Comment (awong-dev) [inline Rakefile]: Interesting...my version of jekyll doesn't have it. I must be out of date.

**Comment by awong-dev** [comment]
LGTM w/o caveat.

I'm still uncertain about `destination: ./_site/roadrunner` but this PR has been outstanding too long and it's less harmful to have that possible mistake checked in than leaving it open.

**Pull request merged**